tests/EndToEnd/Bugs/errors.twig

```diff
@@ -0,0 +1,7 @@
+{% if foo is defined %}{{ foo }}{% endif %}
```

Comment:
A defined check should prevent an "undefined variable" error.

Reply:
I think this is solved by https://github.com/twigstan/twigstan/pull/40